crates/sui-json-rpc-types/src/sui_object.rs

```diff
@@ -48,8 +48,10 @@ pub struct SuiObjectData {
     /// Base64 string representing the object digest
     pub digest: ObjectDigest,
     /// The type of the object. Default to be None unless SuiObjectDataOptions.showType is set to true
+    #[schemars(with = "Option<String>")]
+    #[serde_as(as = "Option<DisplayFromStr>")]
```

Comment:
TIL. I was trying to use `ObjectType` but encountered some errors which could have been resolved by these two macros